assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -0,0 +1,275 @@
+
+  const actPageOptions = actPageList.map(actPage => ({ label: actPage.title.raw, value: actPage.id }));
+
+  const renderEditInPlace = () => (takeActionPageSelected ?
+    <TakeActionBoxoutFrontend {...attributes} /> :
+    <section
+      className={`cover-card action-card ${className || ''}`}
+      style={{
+        backgroundImage: `linear-gradient(180deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0)), url(${imageUrl})`
+      }}
+    >
+      {imageUrl && <img src={imageUrl} alt={imageAlt} />}
+      {tags && tags.map(tag => (
+        <a
+          key={tag.name}
+          className="cover-card-tag"
+          href={tag.link}
+        >
+          <span aria-label="hashtag">#</span>{tag.name}
+        </a>
+      ))}
+      <RichText
+        tagName='div'
+        className='cover-card-heading'
+        placeholder={__('Enter title', 'planet4-blocks-backend')}
+        value={title}
+        onChange={toAttribute('title')}
+        disabled={true}
+        keepPlaceholderOnFocus={true}
+        withoutInteractiveFormatting
+        multiline='false'
+        allowedFormats={[]}
+      />
+      <RichText
+        tagName='p'
+        className='cover-card-excerpt'
+        placeholder={__('Enter description', 'planet4-blocks-backend')}
+        value={excerpt}
+        onChange={toAttribute('excerpt')}
+        disabled={takeActionPageSelected}
+        keepPlaceholderOnFocus={true}
+        withoutInteractiveFormatting
+        allowedFormats={['core/bold', 'core/italic']}
+      />
+      <RichText
+        tagName='div'
+        className="btn btn-action btn-block cover-card-btn"
+        placeholder={__('Button text', 'planet4-blocks-backend')}
+        value={linkText}
+        onChange={toAttribute('linkText')}
+        disabled={takeActionPageSelected}
+        keepPlaceholderOnFocus={true}
+        withoutInteractiveFormatting
+        multiline="false"
+        allowedFormats={[]}
+      />
+    </section>
+  );
+
+  const renderSidebar = () => (
+    <Fragment>
+      <InspectorControls>
+        <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+          <SelectControl
+            label={__('Select Take Action Page:', 'planet4-blocks-backend')}
+            value={take_action_page}
+            options={[
+              { label: __('None', 'planet4-blocks-backend'), value: 0 },
+              ...actPageOptions,
+            ]}
+            onChange={page => updateTakeActionPage(parseInt(page))}
+          />
+          <p className='help'>{__('Or customise your Take Action Boxout:', 'planet4-blocks-backend')}</p>
+
+          <URLInput
+            label={__('Custom link', 'planet4-blocks-backend')}
+            placeholder={__('Enter custom link', 'planet4-blocks-backend')}
+            value={link}
+            onChange={toAttribute('link')}
+            disabled={takeActionPageSelected}
+          />
+          <CheckboxControl
+            label={__('Open in a new tab', 'planet4-blocks-backend')}
```

Comment:
The disabled state of this input is reaaally subtle (at least on my screen), it might make sense in a future iteration to change the css to make it more obvious, or even not display it if it's disabled (displaying disabled url and tags fields make sense to see the information in there, but for this checkbox I'm not sure :shrug: )